ROADMAP.md

```diff
@@ -0,0 +1,51 @@
+# Minikube Roadmap
+This document contains the goals, plans, and priorities for the minikube project.
+
+## Goals
+The primary goal of minikube is to make it simple to run Kubernetes on your laptop, both for getting started and day-to-day development workflows.
+Here are some specific features that align with our goal:
+* Single command setup and teardown UX.
+* Unified UX across OSes
+* Minimal dependencies on third party software.
+* Minimal resource overhead.
+* Replace any other alternatives to local cluster deployment.
```

Comment:
nit: Is it just me or does `all other alternatives` or `any other alternative` sound better?
I'd prefer the first one.